images/disks-images-provider/entrypoint.sh

```diff
@@ -63,7 +57,8 @@ mkdir -p /host/tmp/hostImages/mount_hp/test
 # When the host is ubuntu, by default, selinux is not used, so chcon is not necessary.
 # If selinux tag is set, use chcon to change /hostImages privileges.
 if [ ${SELINUX_TAG:0:1} != "?" ]; then
-    chcon -Rt svirt_sandbox_file_t /host/tmp/hostImages/mount_hp
+    chcon -Rt svirt_sandbox_file_t /host/tmp/hostImages
```

Comment:
@akalenyu I see that previously in this file the `hostImages` directory is being created under root `/`, is that ok that you are doing the `chcon` on `/host/tmp/..`? same for the `local-storage`

Reply:
Ok after looking at the manifest its much more clear whats going on with all these directories